### PR TITLE
Interactive feature

### PR DESCRIPTION
**Pull request by zewdu444**
# To Do list: interactive list feature

- [x] Implement functionality for to clean completed tasks 
- [x] Interactive checkbox strike through added
- [x] Distribution file has been generated
- [x]  Readme file has been updated
- [x] Live demo deployed

### CODE REVIEW

File: src/module/updateTask.js [hunk 1]
```diff
@@ -6,4 +6,11 @@ const updateTask = (index, description) => {
...
-export default updateTask;
\ No newline at end of file
+
+const completeTask = (index, completed) => {
+  index = tasks.findIndex((object) => object.index === index);
+  if (index !== -1) {
+    tasks[index].completed = completed;
+  }
+};
+export { updateTask, completeTask };
```
Comment (zieeco) [inline src/module/updateTask.js]: - Hey, great job using the findIndex method here, however, according to the project requirement, you should use the `filter` method. please, kindly refactor to meet the project requirement below:

Screenshot from 2023-01-19 17-56-42

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @zewdu444 ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/module/updateTask.js (hunk 1 above)
Comment (zewdu444) [reply src/module/updateTask.js]: @zieeco thanks changed find index method to filter as per your recommendation.

**Review by alexander16108 — APPROVED** [review]
Hi @zewdu444 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**